core/src/main/java/org/infinispan/configuration/parsing/Parser.java

```diff
@@ -1253,19 +1253,25 @@ private void parseSharedStateCacheElement(XMLExtendedStreamReader reader, Elemen
       }
    }
 
-   private void parseBackups(XMLExtendedStreamReader reader, ConfigurationBuilder builder) throws XMLStreamException {
+   private void parseBackups(XMLExtendedStreamReader reader, ConfigurationBuilderHolder holder) throws XMLStreamException {
+      ConfigurationBuilder builder = holder.getCurrentConfigurationBuilder();
       // If backups is present then remove any existing backups as they were added by the default config.
       builder.sites().backups().clear();
+      for (int i = 0; i < reader.getAttributeCount(); i++) {
+         String value = reader.getAttributeValue(i);
+         Attribute attribute = Attribute.forName(reader.getAttributeLocalName(i));
+         if (attribute == Attribute.CONFLICT_RESOLVER) {
```

Comment:
For usability it's probably worth exposing an ENUM for the built in resolution policies like "PreferNullConflictResolver". For CR we do this by loading the class if the provided value does not equal one of the provided enum values:

https://github.com/infinispan/infinispan/blob/8150e1dabe7e95167088b0301a202984f1d19748/core/src/main/java/org/infinispan/configuration/parsing/Parser.java#L1289-L1294